stable/admin/values.yaml

```diff
@@ -114,7 +117,7 @@ admin:
   # --set admin.configFiles.nuodb\\.lic=<BASE64-TEXT-HERE>
```

Comment:
Is it worth handling the `dot` in the pathname specially?
Perhaps converting any `underscore` char in the pathname to a `dot` - or similar?

Reply:
Not really.